src/coreclr/nativeaot/Runtime.Base/src/System/Runtime/ExceptionHandling.wasm.cs

```diff
@@ -117,6 +120,48 @@ private static void HandleExceptionWasmFault(void* pShadowFrame, DispatchData* p
             DispatchContinueSearch(exception, pDispatchData);
         }
 
+        // This handler is called by codegen for exceptions that escape from RPI methods (i. e. unhandled exceptions).
+        //
+        private static void HandleUnhandledException(object exception)
```

Comment:
Note that this handler is not a "dispatcher" like the methods above it. This is an intentional simplification - it would have been possible to define a new dispatcher kind (essentially a filter without `catch`) and use it instead of this, and it would have been that little bit better size-wise. It would also have been possible to add the `try` wrapping much later and avoid pessimizations associated with having EH in code. I decided against both - RPI methods are rare, thus both CQ value and testability of such strategies is low.